extensions/smallrye-health/deployment/src/test/java/io/quarkus/smallrye/health/test/DeprecatedHealthOpenAPITest.java

```diff
@@ -0,0 +1,40 @@
+package io.quarkus.smallrye.health.test;
+
+import org.hamcrest.Matchers;
+import org.jboss.shrinkwrap.api.asset.EmptyAsset;
+import org.jboss.shrinkwrap.api.asset.StringAsset;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.test.QuarkusUnitTest;
+import io.restassured.RestAssured;
+
+@Deprecated(since = "3.14", forRemoval = true)
```

Comment:
Well, the test can stay right, we just need to update the config? This test needs to be updated nevertheless. Just mentioning it so it's not removed.

Reply:
No, the test will get dropped. I added another test with the updated config (but GitHub shows it to you the other way around).

Reply:
I see now.